eclair-node/src/main/scala/fr/acinq/eclair/Plugin.scala

```diff
@@ -18,12 +18,13 @@ package fr.acinq.eclair
 
 import java.io.File
 import java.net.{JarURLConnection, URL, URLClassLoader}
-
 import grizzled.slf4j.Logging
-
 import scala.util.{Failure, Success, Try}
 
 trait Plugin {
+  // Set of LightningMessage tags this plugin is interested in and a Feature bit it advertizes through Init message
```

Comment:
nit: this comment will become obsolete as soon as we add more data to the `PluginParams` structure.
Why not move these descriptions inside `PluginParams`? The name of the field here is self-explanatory and shouldn't need any comment.